src/main/java/com/github/jcrfsuite/util/CrfSuiteLoader.java

```diff
@@ -111,9 +111,11 @@ private static void loadCrfSuiteSystemProperties() {
 	  
     private static ClassLoader getRootClassLoader() {
         ClassLoader cl = Thread.currentThread().getContextClassLoader();
+        /* We want to keep the tacokit classloader
         while (cl.getParent() != null) {
```

Comment:
Great job to fix this issue!
One question according to read the comment https://github.com/Talend/jcrfsuite/blob/af1d3735a7670d9215e5cfa8ac6c062db638f8b5/src/main/java/com/github/jcrfsuite/util/CrfSuiteLoader.java#L222 does it mean if we have more This components in same pipeline, there will be an issue if the lib is not in the parent class loader?

Reply:
If I recall correctly, each component have its own classloader.
I will double check this usecase to be sure this is working correctly.

Reply:
ah you are right, classloader for per plugin/family, not per component

Reply:
Checked and ok.